tasklist/hi.json

```
@@ -0,0 +1,520 @@
+    " INTEGER " : " इंटेगर " ,
+    " LONG " : " लंबी " ,
+    " DOUBLE " : " डबल " ,
+    " DATE " : " दिनांक " ,
+    " STRING " : " स्ट्रिंग " ,
+    " AUTH_LOGOUT_SUCCESSFUL " : " लॉगआउट सफल " ,
+    " AUTH_LOGOUT_THANKS " : " आप Camunda आज का उपयोग कर के लिए धन्यवाद एक महान {{dayContext}} है।! " ,
+    " AUTH_DAY_CONTEXT_MORNING " : " सुबह " ,
+    " AUTH_DAY_CONTEXT_DAY " : " दिन " ,
+    " AUTH_DAY_CONTEXT_AFTERNOON " : " दोपहर " ,
+    " AUTH_DAY_CONTEXT_EVENING " : " शाम " ,
+    " AUTH_DAY_CONTEXT_NIGHT " : " रात " ,
+    " AUTH_DAY_CONTEXT_WEEKEND " : " सप्ताहांत " ,
+    " AUTH_FAILED_TO_DISPLAY_RESOURCE " : " संसाधन प्रदर्शित करने में विफल " ,
+    " AUTH_AUTHENTICATION_FAILED " : " प्रमाणीकरण विफल आपका सत्र समाप्त हो हो सकता है, यदि आप प्रवेश करने की जरूरत है।। " ,
+    " PAGE_LOGIN_ERROR_MSG " : " गलत क्रेडेंशियल्स, लॉक किए गए उपयोगकर्ता या आवेदन करने के लिए अनुपलब्ध अधिकार " ,
+    " PAGE_LOGIN_USERNAME " : " प्रयोक्ता नाम " ,
+    " PAGE_LOGIN_PASSWORD " : " पासवर्ड " ,
+    " PAGE_LOGIN_PLEASE_SIGN_IN " : " में कृपया लॉग " ,
+    " PAGE_LOGIN_SIGN_IN_ACTION " : " लॉग इन " ,
+    " PAGE_LOGIN_FAILED " : " प्रवेश विफल " ,
+    " SIGN_OUT_ACTION " : " लॉग आउट करें " ,
+    " MY_PROFILE " : " मेरी प्रोफ़ाइल " ,
+    " DIRECTIVE_ENGINE_SELECT_TOOLTIP " : " ।। आप आप यहाँ प्रक्रिया इंजन का चयन कर सकते हैं चल रहा है कई इंजन हैं तो डेटा इस आवेदन में दिखाया गया चयनित इंजन केवल पर आधारित है " ,
+    " DIRECTIVE_ENGINE_SELECT_STATUS_NOT_FOUND " : " नहीं मिला " ,
+    " DIRECTIVE_ENGINE_SELECT_MESSAGE_NOT_FOUND " : " प्रक्रिया इंजन आप पहुँचने का प्रयास कर रहे हैं मौजूद नहीं है " ,
+    " DIRECTIVE_INPLACE_TEXTFIELD_ERROR_MSG " : " आपको संपादन योग्य होने के लिए संदर्भ की एक संपत्ति निर्दिष्ट करनी चाहिए " ,
+    " PAGES_STATUS_SERVER_ERROR " : " सर्वर त्रुटि " ,
+    " PAGES_MSG_SERVER_ERROR " : " सर्वर में आंतरिक त्रुटि सूचना पेज या लॉगिन और आवेदन के बाहर ताज़ा करने के लिए प्रयास करें।। " ,
+    " PAGES_STATUS_REQUEST_TIMEOUT " : " अनुरोध टाइमआउट " ,
+    " PAGES_MSG_REQUEST_TIMEOUT " : " । आपका अनुरोध का समय समाप्त पृष्ठ को ताज़ा करने की कोशिश करें। " ,
+    " PAGES_STATUS_ACCESS_DENIED " : " प्रवेश निषेध " ,
+    " PAGES_MSG_ACCESS_DENIED " : " आप {{missingAuths}} में अनधिकृत कर रहे हैं। " ,
+    " PAGES_MSG_ACCESS_DENIED_RESOURCE_ID " : " आप {{permissionName}} {{ResourceName}} {{RESOURCEID}} में अनधिकृत कर रहे हैं। " ,
+    " PAGES_MSG_ACTION_DENIED " : "  एक क्रिया का कार्यान्वयन सर्वर द्वारा नहीं दी गई है पृष्ठ को ताज़ा करने का प्रयास करें " ,
+    " PAGES_STATUS_NOT_FOUND " : " नहीं मिला " ,
+    " PAGES_MSG_NOT_FOUND " : "  एक आपके द्वारा अनुरोधित संसाधन नहीं पाया जा सका " ,
+    " PAGES_STATUS_COMMUNICATION_ERROR " : " संचार त्रुटि " ,
+    " PAGES_MSG_COMMUNICATION_ERROR " : " इस अनुप्रयोग में अनपेक्षित {{स्थिति}} सर्वर से प्रतिक्रिया प्राप्त पेज या लॉगिन और आवेदन के बाहर ताज़ा करने के लिए प्रयास करें " ,
+    " PAGES_MSG_ENGINE_NOT_EXISTS " : " जिस प्रक्रिया इंजन को आप एक्सेस करने का प्रयास कर रहे हैं वह मौजूद नहीं है , " ।
```

The delimiter is wrong and the file no longer a valid JSON

```suggestion
    " PAGES_MSG_ENGINE_NOT_EXISTS " : " जिस प्रक्रिया इंजन को आप एक्सेस करने का प्रयास कर रहे हैं वह मौजूद नहीं है , ",
```